docs/configuration.md

```diff
@@ -44,17 +44,17 @@ Mandatory Parameters are marked as **Required**.
 | `exchange.sandbox` | false | Use the 'sandbox' version of the exchange, where the exchange provides a sandbox for risk-free integration. See [here](sandbox-testing.md) in more details.
 | `exchange.key` | '' | API key to use for the exchange. Only required when you are in production mode.
 | `exchange.secret` | '' | API secret to use for the exchange. Only required when you are in production mode.
-| `exchange.pair_whitelist` | [] | List of currency to use by the bot. Can be overrided with `--dynamic-whitelist` param.
-| `exchange.pair_blacklist` | [] | List of currency the bot must avoid. Useful when using `--dynamic-whitelist` param.
+| `exchange.pair_whitelist` | [] | List of currency to use by the bot.
```

Comment:
I guess we should still mention that `pairlist.method=dynamic` will override this?

Reply:
Oops, I thought it was a link to a section with description of the deprecated option. Restored it.

Reply:
I have a better idea I guess. See below or in a sep. issue describing the proposal (or a PR with implementation).

Reply:
apart from that (see below) - both this and the next line are oddly worded... (i've just seen that now)

we have pairs in this list, not currencies ... 
maybe 
`List of pairs to check for potential trades.`
`List of pairs to absolutely avoid.`
?

Reply:
Yes, right.

Reply:
But not 'for potential trades' since whitelist is used in live trades as we know... ;-)

Reply:
Fixed, pls review.